src/main/msp/msp.c

```diff
@@ -3296,6 +3297,12 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
 #endif
         break;
 
+#ifdef USE_RANGEFINDER
+            rangefinderConfigMutable()->rangefinder_hardware = sbufReadU8(src);
+#else
+            sbufReadU8(src);        // rangefinder hardware
+#endif
```

Comment:
We need update this in configurator as well.
```suggestion
#ifdef USE_RANGEFINDER
        rangefinderConfigMutable()->rangefinder_hardware = sbufReadU8(src);
#else
        sbufReadU8(src);        // rangefinder hardware
#endif
```